web/themes/custom/westy/templates/content/node--notification--featured.html.twig

```diff
@@ -0,0 +1,40 @@
+{# Set Notification Type #}
+{% if node.field_notification_serverity.value == 20 %}
```

Comment:
@radmosley , I don't see where the type is used anywhere. This should be removed if not in use.